utilities/rpp-performancetests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -1955,4 +1955,4 @@ int main(int argc, char **argv)
     free(outputi8);
 
     return 0;
-}
+}
```

Comment:
There are all the following files touched in this PR that show as one line changed. Please revert.
```
utilities/rpp-performancetests/HOST_NEW/Tensor_host_pkd3.cpp
utilities/rpp-performancetests/HOST_NEW/Tensor_host_pln1.cpp
utilities/rpp-performancetests/HOST_NEW/Tensor_host_pln3.cpp
utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp
utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp
utilities/rpp-unittests/HOST_NEW/Tensor_host_pln1.cpp
utilities/rpp-unittests/HOST_NEW/Tensor_host_pln3.cpp
```

Reply:
I tried copying all these files directly from TOT develop, but still it showing the line addition there

Reply:
Resolved